Adyen/UI/List/ListItem.swift

```diff
@@ -28,9 +28,6 @@ public class ListItem: FormItem {
     /// The trailing text of the item.
     public var trailingText: String?
     
-    /// A boolean value indicating whether a disclosure indicator should be shown in the item's cell.
-    public var showsDisclosureIndicator: Bool
```

Comment:
why is this removed, its needed for the gift card elements that are not clickable, so the disclosure needs to be .none in this case

Reply:
This is turns `chevron` on and off..

Reply:
yes, its needed to hide the chevron for gift card items

Reply:
but if we remove it - there will be no chevron on Drop-in, and it is fine [as we discussed above](https://github.com/Adyen/adyen-ios/pull/535#discussion_r649767666)

Reply:
but its in the design, right? if the designers are okay with it, then its fine.

Reply:
i haven't talked to design yet. i will